app/src/main/java/com/pvp/app/ui/screen/filter/Filter.kt

```diff
@@ -0,0 +1,132 @@
+package com.pvp.app.ui.screen.filter
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Card
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.compose.runtime.LaunchedEffect
+import com.pvp.app.model.SportActivity
+
+@Composable
+fun ActivitiesFilter(
+    model: FilterViewModel = hiltViewModel()
+) {
+    val activities: List<String> = SportActivity.values().map { it.title }
```

Comment:
Could be transfered out of the activities filter component to make it singleton (computed only once) or into `LaunchedEffect` block to make it recompute only after user value changes, but not every time the component is rerendered.